<launch>
    <!-- STEP 1: START CAMERA DRIVER -->
    <!-- Includes from other ROS packages-->
    <include file="$(find shape_registration)/launch/start_azure_rectifiers.launch"/>


    <!-- STEP 2: RUN THE NODE WHICH WILL CROP THE ROI AND GENERATE THE POINTCLOUD -->
    <node pkg="shape_registration" type="ROI_cropping_node" name="point_cloud_generator">
        <remap from="/camera_info" to="/k4a/depth_to_rgb/camera_info"/>
        <remap from="/depth_to_rgb_image" to="/k4a/depth_to_rgb/image_rect"/>
        <remap from="/rgb_image" to="/k4a/rgb/image_rect_color"/>
    </node>
</launch>
